tools/postinstall/apply-patches.js

```diff
@@ -14,7 +14,7 @@ const chalk = require('chalk');
  * Version of the post install patch. Needs to be incremented when
  * existing patches or edits have been modified.
  */
-const PATCH_VERSION = 6;
+const PATCH_VERSION = 8;
```

Comment:
What happened to 7!?

Did we skip it because seven eight nine?   <sup>[explanation link](https://explainthejoke.com/2013/01/24/7-8-9/) for those who haven't heard this joke

Reply:
hahah. I was expecting that you notice. I messed up the patch in my first PR pushes, and to make it run in the fixup I needed to increment it twice.

Reply:
Sounds good, just wanted to confirm.

Reply:
yeah, thanks for raising this. Could have totally also been by accident from local testing.